### ISSUE TITLE
[bug] testID does not apply in builds

### ISSUE THREAD

**Issue by christianalares** [tamagui/tamagui#354]
**Describe the bug**
`data-testid` is not applied to Input element

**To Reproduce**
* Create a new project:
```
npm create tamagui-app@latest
```
* Replace `apps/next/pages/index.tsx` with the following:
```javascript
// import { HomeScreen } from 'app/features/home/screen'
// export default HomeScreen

import { Input } from 'tamagui'

const Page = () => {
  return <Input value="hello world" testID="testing" className="testing-classname" />
}

export default Page
```
* Build:
```
cd apps/next && yarn build
```

* Serve the app:
```
cd apps/next && yarn start
```

* Navigate to `http://localhost:3000/` and inspect the input. Should look something like this: 
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/893819/201340831-d937a8d9-7d31-4a77-8f7f-1312651bf282.png">

**System Info (please complete the following information):** 

Output of `npx envinfo --system --npmPackages --binaries --browsers`

**Tamagui info**
```
System:
  OS: macOS 13.0
  CPU: (10) arm64 Apple M1 Max
  Memory: 2.96 GB / 64.00 GB
  Shell: 5.8.1 - /bin/zsh
Binaries:
  Node: 16.17.0 - /usr/local/bin/node
  Yarn: 3.2.3 - /usr/local/bin/yarn
  npm: 8.15.0 - /usr/local/bin/npm
Browsers:
  Chrome: 107.0.5304.110
  Firefox Developer Edition: 104.0
  Safari: 16.1
  Safari Technology Preview: 15.4
npmPackages:
  react-native: 0.0.0 => 0.0.0
  ```

**Misc**
The `className` attribute is also missing in the attributes

**Comment by natew** [comment]
Fixed in [96f881c](https://github.com/tamagui/tamagui/pull/356/commits/96f881c283b6cc41797b5ff2f22041794438664b)

**Comment by natew** [comment]
In beta 215, reopen if still an issue
